astropy/stats/funcs.py

```diff
@@ -538,8 +538,7 @@ def poisson_conf_interval(n, interval='root-n', sigma=1, background=0,
     -----
 
     The "right" confidence interval to use for Poisson data is a
-    matter of debate. The CDF working group `recommends
-    <https://www-cdf.fnal.gov/physics/statistics/notes/pois_eb.txt>`_
```

Comment:
This link is now behind a Fermilab login wall. 🤷

Reply:
https://web.archive.org/web/20210222093249/https://www-cdf.fnal.gov/physics/statistics/notes/pois_eb.txt

Reply:
Awesome. Thanks!